src/dataset/DatasetRegistry.py

```diff
@@ -1,24 +1,18 @@
-from src.dataset.FFHQDataset import FFHQDataset
+from src.dataset.FFHQDataset import FFHQDataset, FFHQ_DEF_NAME
 from src.dataset.Dataset import Dataset
+from src.core.Registry import Registry
 
 
-class DatasetRegistry:
+class DatasetRegistry(Registry):
     """
-    Static class used for initialization and storing of all subclasses of the
+    Static class implementing the abstract Registry class
+    used for initialization and storing of all subclasses of the
     Dataset class.
-    """
-
-    _DATASETS = None
 
-    @staticmethod
-    def init_registry():
-        """
-        Static method to initialize `DATASETS` with Dataset subclasses.
-        Calls the constructor for each dataset with predefined parameters.
+    * If more Datasets are implemented they should be added in this function.
```

Comment:
What function?

"If more Datasets are implemented they must be manually added to the internal storage of this class." perhaps